src/plugins/intel_cpu/src/utils/shape_inference/static_dimension.hpp

```diff
@@ -27,6 +27,11 @@ class StaticDimension {
     /// \param dimension Value of the dimension.
     StaticDimension(value_type dimension);
 
+    /// \brief Construct a static dimension.
+    /// \param ldimension Value of the dimension (must be equal to ldimension)
```

Comment:
Can you explain variable name `ldimension` in comment `lower bound of dimension`?
In `()` is mentioned that this variable must be equal to `ldimension`, itself. Is that ok? shoud `udimension`

Check also description of `udimension`